course_access_groups/permissions.py

```diff
@@ -28,26 +29,31 @@ def is_organization_staff(user, course):
     """
     Helper to check if the user is organization staff.
 
+    Q: What if a course has two orgs?
+    A: No problem. This function raises `MultipleObjectsReturned`
+
     :param user: User to check access against.
     :param course: The Course or CourseOverview object to check access for.
 
     :return: bool
-
-    TODO: Handle single-site setups in which organization is not important
-    TODO: What if a course has two orgs? data leak I guess?
     """
-
     if not user.is_active:
         # Checking for `user.is_active` again. Better to be safe than sorry.
         return False
 
-    # Same as organization.api.get_course_organizations
-    course_org_ids = OrganizationCourse.objects.filter(
-        course_id=str(course.id),
-        active=True,
-    ).values('organization_id')
+    try:
+        course_organization = get_organization_by_course(course_id=course.id)
+    except Organization.DoesNotExist:
+        # Safely handle the exception errors by assuming the user is not a staff.
+        return False
+    except MultipleObjectsReturned:
+        log.exception(
+            'Course Access Group: This module expects a one:one relationship between organizations and course. '
+            'Raised by course ({course_id})'.format(course_id=course.id)
+        )
```

Comment:
`format` is not always safe for `log`, let the logger handle formatting to ensure that the log don't fail due to encoding or `str` errors:

Also use `log.exception` less when possible.

```suggestion
        log.warning(
            'Course Access Group: This module expects a one:one relationship between organizations and course. '
            'Raised by course (%s)', course.id
        )
```